dips/dip-3.md

```diff
@@ -10,40 +10,34 @@ created: 06/09/2020
+Writeset transactions are a special type of multi-sig transactions that prescribes an arbitrary state change. Unlike normal transactions, which must follow specified Diem Framework rules, WriteSet transactions simply describe a state change rather than go through the Diem Framework modules. This type of transaction is only needed for situations such as updating immutable stdlib libraries due to critical bugs or reversing systematic fraudulent transactions.
 
 The use of WriteSet transactions follows the spirit of transparency — WriteSet transactions allow an arbitrary change to the state of the blockchain without altering the history of past transactions.
-Even though WriteSet transactions prescribe arbitrary state change, they are authenticated with the same multi-sig mechanism as normal transactions which means both the change and multi-sig are transparent and visible on-chain.
+Even though WriteSet transactions prescribe an arbitrary state change, they are authenticated with the same multi-sig mechanism as normal transactions which means both the change and multi-sig are transparent and visible on-chain.
 
 ### Genesis transactions
-Genesis transactions can be either normal or WriteSet transactions. As the name suggests, the process of applying such a transaction is similar to how the very first genesis transaction works. The genesis transaction applies on top of a specific snapshot of a Diem blockchain (might be empty). DN is responsible for picking the appropriate snapshot and building the genesis transactions. Users of Diem including validators use the genesis transaction to authenticate the blockchain. If a validator fails to acquire the correct genesis transaction, it will vote on an irrelevant blockchain, but not otherwise impact the system. If a user of the system acquires the incorrect genesis transaction, they will see incorrect state.
+Genesis transactions can be either normal or WriteSet transactions but without signatures. As the name suggests, the process of applying such a transaction is similar to how the very first genesis transaction works. The genesis transaction applies on top of a specific snapshot of a Diem blockchain (might be empty). DN is responsible for picking the appropriate snapshot and building the genesis transactions. Users of Diem, including validators, use the genesis transaction to authenticate the blockchain. If a validator fails to acquire the correct genesis transaction, it will vote on an irrelevant blockchain, but not otherwise impact the system. If a user of the system such as a full node or blockchain observer acquires the incorrect genesis transaction, they will see an incorrect state.
```

Comment:
```suggestion
Genesis transactions can be either normal or WriteSet transactions without the typical corresponding signatures. As the name suggests, the process of applying such a transaction is similar to how the very first genesis transaction works. The genesis transaction applies on top of a specific snapshot of a Diem blockchain (might be empty). DN is responsible for picking the appropriate snapshot and building the genesis transactions. Users of Diem, including validators, use the genesis transaction to authenticate the blockchain. If a validator fails to acquire the correct genesis transaction, it will vote on an irrelevant blockchain, but not otherwise impact the system. If a user of the system such as a full node or blockchain observer acquires the incorrect genesis transaction, they will see an incorrect state.
```